server/pages/index.jade

```diff
@@ -110,25 +110,45 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
-		if user
+		if supportUser
+			script.
+				window.supportUser = {
+					readyScripts: []
+				};
+		if user && !supportUser
 			script(type='text/javascript')!='var currentUser = ' + sanitize.jsonStringifyForHtml( user )
 		if app
 			script(type='text/javascript')!='var app = ' + sanitize.jsonStringifyForHtml( app )
-		if initialReduxState
+		if initialReduxState && !supportUser
```

Comment:
`initialReduxState` here is not data from localStorage. It is API response data that has been used to initially render the requested page on the server, and is here being passed to the client so that the client will not need to fetch data and re-render. It is only ever used for logged-out pages. I believe this behaviour needs to be consistent whether the support user is active or not, so we probably don't want this check added here.

Thanks for the ping @gwwar